### ISSUE TITLE
Locate object only shows the most recent 25 items in game, it can be impossible to check rares.

### ISSUE THREAD

**Issue by Lilithelle** [torilmud/issues#774]
In this instance someone has 25 sacred totems in a bag, now no one can locate to see whether it is up or not.  It allows people to block others from checking rares easily, which they just need to log them out if they want to check the rare.  For other items you might want to locate there are just a lot of them in game.  I suggest making locate object show the most recent 25 items, and the least recent 25 items as well to get around this.

sacred totem of the snow trolls in some sort of container.
sacred totem of the snow trolls in some sort of container.
sacred totem of the snow trolls in some sort of container.
sacred totem of the snow trolls in some sort of container.
sacred totem of the snow trolls in some sort of container.
sacred totem of the snow trolls in some sort of container.
sacred totem of the snow trolls in some sort of container.
sacred totem of the snow trolls in some sort of container.
sacred totem of the snow trolls in some sort of container.
sacred totem of the snow trolls in some sort of container.
sacred totem of the snow trolls in some sort of container.
sacred totem of the snow trolls in some sort of container.
sacred totem of the snow trolls in some sort of container.
sacred totem of the snow trolls in some sort of container.
sacred totem of the snow trolls in some sort of container.
sacred totem of the snow trolls in some sort of container.
sacred totem of the snow trolls in some sort of container.
sacred totem of the snow trolls in some sort of container.
sacred totem of the snow trolls in some sort of container.
sacred totem of the snow trolls in some sort of container.
sacred totem of the snow trolls in some sort of container.
sacred totem of the snow trolls in some sort of container.
sacred totem of the snow trolls in some sort of container.
sacred totem of the snow trolls in some sort of container.
sacred totem of the snow trolls in some sort of container.

**Comment by Dartan82** [comment]
Or just make Rashnik's totem !locate...

**Comment by Lilithelle** [comment]
Well that leaves every other rare in the game like orb of annihilation.  However, locate object is also how I tell if a zone is done, I have a whole file of locates to test which have and haven't been done.  Items that were once rarely in game now exist in large numbers.Lilithelle    On Thursday, July 23, 2020, 09:29:06 a.m. PDT, Dartan82 <notifications@github.com> wrote:  
 
 


Or just make Rashnik's totem !locate...

**Comment by Dartan82** [comment]
annihilation never has an issue checking if it's in game or not

this issue is moot if snow totem is !locate.

**Comment by jokkmeister** [comment]
I think all rares should be !locate. That would solve alot of problems as well as giving people something to do when they actually have to run and check for things instead of just casting locate object.

**Comment by Dartan82** [comment]
I find it amusing the writer of UFC and GD is asking for it to be easier to locate rares.

**Comment by gonoph** [comment]
Major zones that don’t repop with gear should have an in game way to see if they’ve been done.

1.  Make rares !locate, including zones that have several popular gear vs non-popular rare loads.
2. In game display of zones’s reboot only % of gear:mobs left

Would resolve:

1. Won’t need to Use locate To see if zones are done.
2. Prevents “gaming the system” by ignoring zones that don’t yield “popular” rare gear in a boot.
3. Prevents folks from racing to specific targets
4. Prevents folks from misleading others if a rare loaded
5. Ensure invasion pops must be checked manually

**Comment by Lilithelle** [comment]
Yes, for UFC I wanted people to have to actually check for rares.  When I write zones I try to pay close attention to nolocate.  You do want most items to be locatable, people do lost items, drop them, forget them etc.  It is good if they can locate to find them.
Gonoph raises a good point, there are zones like CPV you need to know what loaded, you could get a group, do they zone and find just 2 items loaded.  It's not worth the time and effort if the load is poor.  Ideally CPV and other zones like magma should be rewritten so instead of getting 2 items or 6 items or anything in between you always get X items or X items +/- 1.  Or always make sure a sought after item loaded, for magma it'd be band of frost, maybe nature's revenge, alit boots, blue dragon tooth earring, etc.  For now, locate object is the best way for CPV, sadly there is no decent way for Magma.
Lilithelle

**Comment by Uagr** [comment]
How about making locate object accept more than 1 variable... example
currently 
cas 'locate o' manta 
receive x25 misty red vial of manta ray poison 

new 
cas 'locate o' cloak manta
uber cloak of manta ray sexiness

**Comment by Lilithelle** [comment]
You can do that now, c 'locate object' cloak.manta
    On Tuesday, August 18, 2020, 09:41:24 a.m. PDT, Uagr <notifications@github.com> wrote:  
 
 


How about making locate object accept more than 1 variable... example
currently
cas 'locate o' manta
receive x25 misty red vial of manta ray poison

new
cas 'locate o' cloak manta
uber cloak of manta ray sexiness

**Comment by Uagr** [comment]
> 
> You can do that now, c 'locate object' cloak.manta On Tuesday, August 18, 2020, 09:41:24 a.m. 

Thanks... never did I every try with a '.', I tried keyword combinations, quotes, etc...maybe could we update the helpfile so the knowledge doesn't get lost. Appreciated

**Comment by vitaladsr** [comment]
> I think all rares should be !locate. That would solve alot of problems as well as giving people something to do when they actually have to run and check for things instead of just casting locate object.

Yaaas

**Comment by shevarash** [comment]
There's a larger discussion about rares here, obviously. But to deal with the immediate problem, perhaps doubling this to 50 items would do the trick.

**Comment by Dartan82** [comment]
Should eliminate the concept of rares.  It's a very 90's concept.
